src/components/popup/tx-item.tsx

```diff
@@ -155,10 +155,10 @@ export const TxItem: React.FC<TxItemProps & BoxProps> = ({ transaction, ...rest
 
   return (
     <Box position="relative" cursor="pointer" {...bind} {...rest}>
-      <Stack alignItems="center" spacing="base-loose" isInline position="relative" zIndex={2}>
+      <Stack alignItems="center" spacing="base-loose" isInline position="relative" zIndex={2}  onClick={() => handleOpenTxLink(transaction.tx_id)}>
```

Comment:
This will not work -- this is what it was before. With the addition of the "increase fee" button, with the onClick at this level (a component that wraps the button), when a user hits the increase fee button, they will activate that drawer, while also leaving the wallet to view the explorer link.

Reply:
Oh yes indeed ! Then we should keep it like before.

Reply:
Don't we need to use `e.stopPropagation()` to solve this event bubbling issue?

Reply:
yeah it's more of a zIndex issue. It can likely be solved with `pointer-events` and `z-index` css changes.

Reply:
Fixed with a e.stopPropagation().